lizmap/modules/admin_api/controllers/repository_rest.classic.php

```diff
@@ -103,4 +105,66 @@ protected function getRepoDetail($rep)
 
         return $rep;
     }
+
+    /**
+     * Handles the creation of a repository based on provided parameters.
+     *
+     * @return object a JSON response object
+     */
+    public function post(): object
+    {
+        $rep = $this->getResponse('json');
+
+        if (!Credentials::handle()) {
+            return Error::setError($rep, 401);
+        }
+
+        $repo = $this->param('repo');
+
+        if (lizmap::getRepository($repo)) {
+            return Error::setError($rep, 400, "The repository '{$repo}' already exists.");
```

Comment:
Error `409 Conflict` 
* https://stackoverflow.com/questions/3825990/http-response-code-for-post-when-resource-already-exists
* https://developer.mozilla.org/fr/docs/Web/HTTP/Reference/Status/409